core/src/main/scala/org/apache/spark/ui/jobs/JobProgressListener.scala

```diff
@@ -431,6 +432,13 @@ class JobProgressListener(conf: SparkConf) extends SparkListener with Logging {
   }
 
   /**
+   * Remove at least (maxRetained / 10) items to reduce friction.
+   */
+  def removedCount(dataSize: Int, retainedSize: Int): Int = {
```

Comment:
`private`? and since it isn't reporting a number that were removed, but a number to remove, maybe `calculateNumberToRemove`? or something?